qiskit/aqua/operators/gradients/hessian_base.py

```diff
@@ -0,0 +1,67 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""The module to compute Hessians."""
+
+from typing import Union
+
+from qiskit.aqua.operators.gradients.circuit_gradients.circuit_gradient \
+    import CircuitGradient
+from qiskit.aqua.operators.gradients.derivative_base import DerivativeBase
+
+
+class HessianBase(DerivativeBase):  # pylint: disable=abstract-method
+    """Base class for the Hessian of an expected value."""
+
+    def __init__(self,
+                 hess_method: Union[str, CircuitGradient] = 'param_shift',
+                 **kwargs):
```

Comment:
Are these `kwargs` needed because the Hessian implementations derive from this? Or what's the reason for it?

Reply:
Yes exactly. More specifically, the parameter shift gradients can be used as finite difference method, whereby the epsilon may be chosen.

Reply:
Hm ok, I would say a user has to pass the ParameterShift object in themselves if they want a specific epsilon for sake of encapsulation but we can do that later.

Reply:
'kwargs' is currently used only for 'epsilon'. But, we leave it 'kwargs' in case other developers extend it.